src/Sylius/Bundle/ApiBundle/test/config/fixtures/foo.yaml

```diff
@@ -0,0 +1,13 @@
+Sylius\Bundle\ApiBundle\Application\Entity\FooSyliusResource:
+    fooSyliusResource3:
+        name: FooSyliusResource3
+
+Sylius\Bundle\ApiBundle\Application\Entity\Foo:
+    foo1:
+        name: Foo0
```

Comment:
can we synchronize this naming?

Reply:
Definitely, we should, but I would like to do that in a separate PR as it requires a little refactor of unit tests